packages/admin/admin-stories/src/docs/components/SplitButton/SplitButton.stories.mdx

```diff
@@ -1,71 +0,0 @@
-import { Meta, Story, Canvas } from "@storybook/addon-docs";
```

Comment:
I removed the Docs on the `SplitButton` together with all stories. But I'm not sure if that's a good idea. Maybe we should keep at least one story for development if we, for some reason, need to change something about the `SplitButton`

Reply:
Resolved with https://github.com/vivid-planet/comet/pull/1623#issuecomment-1926405750